Modelica/Math/package.mo

```diff
@@ -4895,32 +4899,32 @@ A_fud  = [-1, 2, -3;
 
     N       (input) INTEGER
             The number of linear equations, i.e., the order of the
-            matrix A.  N >= 0.
+            matrix A.  N &gt;= 0.
```

Comment:
I would prefer using `&ge;` instead of `&gt;=` and similarly `&le;` instead of `&lt;=`.

Reply:
As far as I understand that would also be a simple search-and-replace.

Reply:
I'm in favor if there is no concern regardig a slightly different feeling of the original and the "Modelica-modified" documentation.

Reply:
I agree with Hans.